assets/screens/shop_screen.onj

```diff
@@ -59,6 +59,105 @@ var personWidget = $PersonWidget {
     ],
 };
 
+/*var cardsWidgetTextChild = $Image {  //This comment as an example for scrollable FlexBox
+                                        //if used, be careful of shopWidget, as it uses some parameters which aren't here now
+    name: "background_img",
+    textureName: "shop_background_no_biome",
+    scaleX: 0.0,
+    scaleY: 0.0,
+    width: worldWidth,
+    height: worldHeight,
+    reportDimensionsWithScaling: true,
+    styles: [
+        {
+            width: 45.0#percent,
+            height: 45.0#percent,
+        }
+    ],
+};
+var scrollBarBackground = $Image {
+
+    name: "scrollbar_background",
+    textureName: "test_scrollbar_background",
+    scaleX: 1.0,
+    scaleY: 1.0,
+    reportDimensionsWithScaling: true,
+};
+var scrollBar = $Image {
+    name: "scrollbar_handle",
+    textureName: "test_scrollbar_background",
+    scaleX: 1.0,
+    scaleY: 1.0,
+    styles: [
+        {
+            width: 5.5#points,
+        }
+    ],
+};
+
+var cardsWidget = $ScrollBox {
+    name: "scroll_box",
+    isScrollDirectionVertical: false,
+    scrollDistance: 2.0,
+    backgroundStretched: false,
+    scrollbarBackgroundName: "scrollbar_background",
+    scrollbarName: "scrollbar_handle",
+    scrollbarSide: "right",
+    styles: [
+        {
+            flexWrap: wrap.wrap,
+            flexDirection: flexDirection.column,
+            justifyContent: justify.spaceBetween,
+            background: "shop_items_background",
+            width: 100.0#percent,
+            height: 69.0#percent,
+            paddingLeft: 3.0#points,
+            paddingRight: 3.0#points,
+            paddingTop: 3.0#points,
+            paddingBottom: 4.0#points,
+            cuttingLeft: 1.75,
+            cuttingRight: 1.75,
+            cuttingTop: 2.0,
+            cuttingBottom: 3.0,
+        }
+    ],
+} children [
+    cardsWidgetTextChild,
+    cardsWidgetTextChild,
+    cardsWidgetTextChild,
+    cardsWidgetTextChild,
+    cardsWidgetTextChild,
+    cardsWidgetTextChild,
+    cardsWidgetTextChild,
+    cardsWidgetTextChild,
+    cardsWidgetTextChild,
+    cardsWidgetTextChild,
+    cardsWidgetTextChild,
+    cardsWidgetTextChild,
+    cardsWidgetTextChild,
+    cardsWidgetTextChild,
+    cardsWidgetTextChild,
+    cardsWidgetTextChild,
+    cardsWidgetTextChild,
+    cardsWidgetTextChild,
+    cardsWidgetTextChild,
+    cardsWidgetTextChild,
+    cardsWidgetTextChild,
+    cardsWidgetTextChild,
+    cardsWidgetTextChild,
+    cardsWidgetTextChild,
+    cardsWidgetTextChild,
```

Comment:
Is that just for testing? If so, it should probably be removed before the pull request is made